nni/common/concrete_trace_utils/concrete_tracer.py

```diff
@@ -1218,12 +1234,12 @@ def clz_getattr_wrapper(obj, attr):
 def concrete_trace(root : Union[torch.nn.Module, Callable[..., Any]],
                    concrete_args: Union[Dict[str, Any], Tuple],
                    *,
-                   use_function_patch: bool = False,
-                   function_patch_backlist: List[str] = [],
+                   use_operator_patch: bool = False,
+                   operator_patch_backlist: List[str] = [],
                    forwrad_function_name: str = 'forward',
                    check_args: Optional[Dict[str, Any]] = None,
-                   autowrap_leaf_function = ConcreteTracer.default_autowrap_leaf_function,
-                   autowrap_leaf_class = ConcreteTracer.default_autowrap_leaf_class,
+                   autowrap_leaf_function = {},
+                   autowrap_leaf_class = {},
```

Comment:
there has risk using [], {} as default value, all function call will share the same [], {}, `None` is better.

Reply:
OK. The latest commit has fixed this issue.